iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -28,11 +28,26 @@ namespace popo
 class SenderPort;
 class ReceiverPort;
 } // namespace popo
+namespace posix
+{
+class UnixDomainSocket;
+class MessageQueue;
+} // namespace posix
 
 using SenderPortType = iox::popo::SenderPort;
 using ReceiverPortType = iox::popo::ReceiverPort;
 
 constexpr char MQ_ROUDI_NAME[] = "/roudi";
+/// @brief The socket is created in the current path if no absolute path is given hence
+///      we need an absolut path so that every application knows where our sockets can
+///      be found.
+#if defined(__APPLE__)
+constexpr char MQ_ROOT_PATH[] = "/tmp";
```

Comment:
@elBoberido No if we use sockets on linux or QNX we have to provide the same path otherwise the sockets are created in the current working directory and all the other apps have no idea where to look for them

Reply:
then I would favour to put this into UnixDomainSocket. Just prepend the name with this and you are done. It's also more save to use, since you cannot forget to set it.

Reply:
It is part of UnixDomainSocket and the sockets can be created with root path (default) or explicitly without a root path.